### ISSUE TITLE
Intermittent integration test failure

### ISSUE THREAD

**Issue by n1k0** [Kinto/kinto.js#408]
This seems to only ever happen with `SERVER=2.1.0 ACTION=test`. 

```
  1) Integration tests Flushed server "before each" hook for "should reject a call to sync() with appropriate message":
     HTTP 0; FetchError: request to http://0.0.0.0:8888/v1/ failed, reason: connect ECONNREFUSED 0.0.0.0:8888
  FetchError: request to http://0.0.0.0:8888/v1/ failed, reason: connect ECONNREFUSED 0.0.0.0:8888
      at ClientRequest.<anonymous> (node_modules/kinto-client/node_modules/isomorphic-fetch/node_modules/node-fetch/index.js:125:11
      at Socket.socketErrorListener (_http_client.js:256:9)
      at emitErrorNT (net.js:1253:8)
```

**Edit:**

May also intermittently fail with latest Kinto master:

```
  1) Integration tests "after all" hook:
     Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
```

**Comment by leplatrem** [comment]
Was this fixed in #411 ?

**Comment by n1k0** [comment]
Yes.
